manifests/profile/neutron/agent.pp

```diff
@@ -1,5 +1,10 @@
 # The profile to set up a neutron agent
 class openstack::profile::neutron::agent {
   include ::openstack::common::neutron
-  include ::openstack::common::ml2::ovs
+
+  case $::openstack::config::neutron_core_plugin {
+    'plumgrid': { include ::openstack::common::plumgrid }
```

Comment:
The core plugin that this parameter refers to is for a setting in neutron.conf and it looks like it is supposed to be 'neutron.plugins.plumgrid.plumgrid_plugin.plumgrid_plugin.NeutronPluginPLUMgridV2', not simply 'plumgrid'